config/locales/en.yml

```diff
@@ -485,6 +485,13 @@ en:
         header: Frequently asked questions
         subheader: Click below for responses to frequently asked questions. Chat with us if you still have questions!
         title: FAQ
+      full_service_home:
+        header: Your money is waiting!
+        subheader1: Does your family make less than $56,000? You might qualify for a refund of up to $3,500.
+        subheader2: Millions of people qualify for government programs to get through hard times. We're offering a free service to help you get your share.
+        subheader3: Trained volunteers are ready to help you! Our single form signs you up for stimulus checks and tax credits. It takes less than an hour to fill out.
+        subheader4: Don't miss out!
+        title: File taxes with help for free!
       home:
```

Comment:
these are the only new strings. the other changes are because i ran i18n-tasks normalize, which i thought was the correct procedure, before committing. seems like we haven't always been doing it.